### DISCUSSION TITLE
Clarification using both www subdomains and custom subdomains

### DISCUSSION THREAD

**Discussion by icolwell** [community/community#22304] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://help.github.com/en/articles/about-supported-custom-domains#www-subdomains" rel="nofollow noopener">This section</a> of the docs states “We strongly recommend that you use a www subdomain” and then goes on to say that “A <em>custom subdomain</em> is a type of subdomain that doesn’t use the standard www subdomain part”</p>
<p>What if I wanted to do both? use a custom subdomain and www at the same time. For example:</p>
<p>“<a href="http://www.blog.example.com" rel="nofollow noopener">www.blog.example.com</a>”</p>
<p>Would I still get the 3 mentioned benefits of using “www” (CDN, DOS protection, etc.) while also being able to use the “blog” subdomain?</p>
<p>From my limited DNS experience, I don’t see how “<a href="http://www.blog.example.com" rel="nofollow noopener">www.blog.example.com</a>” would be any different than “<a href="http://www.example.com" rel="nofollow noopener">www.example.com</a>”, but I don’t know what is happening on the github pages side of things and the documentation isn’t 100% clear.</p>
<p>Any clarification will be greatly appreciated!</p>
<p>Thanks</p>
<p>Edit: Also, for the CNAME file in the github repo is it suggested to put “<a href="http://blog.example.com" rel="nofollow noopener">blog.example.com</a>” or “<a href="http://www.blog.example.com" rel="nofollow noopener">www.blog.example.com</a>”? Are both equivalent? (assuming you have www CNAME setup on your DNS provider too).</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>To my understanding, for the purposes of the documentation, everything before <code>.example.com</code> can be considered the “subdomain”. Because of this, <code>www.blog</code> does not equal <code>www</code> and would not get all or possibly any of the mentioned benefits.</p>
<p>As for what to put in the <code>CNAME</code> file in the repository, you would put the fully-qualified domain name that you wished to use in the file. So if you choose to use <code>[www.blog.example.com](http://www.blog.example.com)</code>, then you would put <code>[www.blog.example.com](http://www.blog.example.com)</code> in your <code>CNAME</code> file.</p>
<p>I hope that helps!</p>
